client-mr/src/main/java/org/apache/hadoop/mapreduce/RssMRConfig.java

```diff
@@ -52,4 +53,15 @@ public class RssMRConfig {
   public static final String RSS_CLIENT_MAX_SEGMENT_SIZE = "mapreduce.rss.client.max.buffer.size";
   public static final long RSS_CLIENT_DEFAULT_MAX_SEGMENT_SIZE = 3 * 1024;
   public static final String RSS_STORAGE_TYPE = "mapreduce.rss.storage.type";
+
+  public static final String RSS_PARTITION_NUM_PER_RANGE = "mapreduce.rss.partitionNum.per.range";
+  public static final int RSS_PARTITION_NUM_PER_RANGE_DEFAULT_VALUE = 1;
+  public static final String RSS_BASE_PATH = "mapreduce.rss.base.path";
```

Comment:
please concerned about the pr about mutli hdfs.

Reply:
The mutli-hdfs pr does not change the conf key of "rss.base.path".

Reply:
> The mutli-hdfs pr does not change the conf key of "rss.base.path".

you should guarantee multi hdfs's pr can work with your implement.

Reply:
In multi hdfs's pr, the interface of ShuffleClientFactory is unchanged.

Reply:
I see that.

Reply:
for client config, there should be a PR to optimize config name. eg,
in coordinator:  rss.client.aaa.bbb
in spark: spark.rss.aaa.bbb
in mr: mapreduce.rss.aa.bbb